sql-server-2014-alwayson-existing-vnet-and-ad/nested/deploy-sql-alwayson.json

```diff
@@ -124,57 +103,58 @@
         }
       }
     },
-
     {
-			"type": "Microsoft.Compute/virtualMachines/extensions",
-			"name": "[concat(parameters('sqlNamePrefix'),'1/CreateCluster')]",
-			"apiVersion": "[variables('apiVersionVMExtensions')]",
-			"location": "[resourceGroup().location]",
-			"dependsOn":[
-				"[concat('Microsoft.Compute/virtualMachines/',parameters('sqlNamePrefix'),'0/extensions/sqlAOPrepare')]"
-			],
-			"properties": {
-				"publisher": "Microsoft.Powershell",
-				"type": "DSC",
-				"typeHandlerVersion": "2.17",
-				"autoUpgradeMinorVersion": false,
-				"settings": {
-					"wmfVersion": "5.0",
-					"modulesUrl": "[variables('createClusterModulesURL')]",
-					"configurationFunction": "[variables('createClusterConfigurationFunction')]",
-					"properties": {
-						"domainName": "[parameters('domainName')]",
-						"clusterName": "[variables('clusterName')]",
-						"sharePath": "[concat('\\\\',parameters('sqlWitnessVMName'),'\\',parameters('sharePath'))]",
-						"nodes": ["[concat(parameters('sqlNamePrefix'),'0')]","[concat(parameters('sqlNamePrefix'),'1')]"],
-						"sqlAlwaysOnEndpointName": "[parameters('sqlAlwaysOnEndpointName')]",
-						"sqlAlwaysOnAvailabilityGroupName": "[parameters('sqlAlwaysOnAvailabilityGroupName1')]",
-						"sqlAlwaysOnAvailabilityGroupListenerName": "[parameters('sqlAlwaysOnAvailabilityGroupListenerName1')]",
-						"sqlAlwaysOnAvailabilityGroupListenerPort": 1433,
-						"lbName": "[parameters('sqlLBName')]",
-						"lbAddress": "[parameters('sqlLBIPAddress')]",
-						"primaryReplica": "[concat(parameters('sqlNamePrefix'),'1')]",
-						"secondaryReplica": "[concat(parameters('sqlNamePrefix'),'0')]",
-						"dnsServerName": "[parameters('dnsServerName')]",
-						"adminCreds": {
-							"userName": "[parameters('adminUserName')]",
-							"password": "PrivateSettingsRef:adminPassword"
-						},
-						"sqlServiceCreds": {
-							"userName": "[parameters('sqlServerServiceAccountUserName')]",
-							"password": "PrivateSettingsRef:sqlServerServiceAccountPassword"
-						}
-					}
-				},
-				"protectedSettings": {
-					"items": {
-						"adminPassword": "[parameters('adminPassword')]",
-						"sqlServerServiceAccountPassword": "[parameters('sqlServerServiceAccountPassword')]"
-					}
-				}
-			}
-		}
-  ],
-    "outputs": {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('sqlNamePrefix'),'1/CreateCluster')]",
+      "apiVersion": "[variables('apiVersionVMExtensions')]",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[concat('Microsoft.Compute/virtualMachines/',parameters('sqlNamePrefix'),'0/extensions/sqlAOPrepare')]"
+      ],
+      "properties": {
+        "publisher": "Microsoft.Powershell",
+        "type": "DSC",
+        "typeHandlerVersion": "2.17",
+        "autoUpgradeMinorVersion": false,
```

Comment:
Any reasons we can't set this to "true"?  That will make the sample last a bit longer...